src/archives/Directory.Build.targets

```diff
@@ -46,16 +46,17 @@
           DependsOnTargets="$(PublishSymbolsToDiskDependsOn)">
     <!-- Collect symbol files and copy to staging directory -->
     <ItemGroup>
-      <FileToArchive Remove="@(FileToArchive)" />
-      <FileToArchive Include="$(ArchiveContentRootPath)**\*.dbg" />
-      <FileToArchive Include="$(ArchiveContentRootPath)**\*.dwarf" />
-      <FileToArchive Include="$(ArchiveContentRootPath)**\*.pdb" />
+      <_SymbolFileToArchive Remove="@(_SymbolFileToArchive)" />
+      <_SymbolFileToArchive Include="$(ArchiveContentRootPath)**\*.dbg" />
+      <_SymbolFileToArchive Include="$(ArchiveContentRootPath)**\*.dwarf" />
+      <_SymbolFileToArchive Include="$(ArchiveContentRootPath)**\*.pdb" />
+      <_SymbolFileToArchive Include="@(SymbolFileToArchive)" />
```

Comment:
Just for my own knowledge (not that anything is wrong here) - where is `SymbolFileToArchive` coming from?

Reply:
Currently, nothing is providing that. It's just the analogous concept of `FileToArchive` but for symbols in case we need it in the future.